content/posts/newsletter-013/index.md

```diff
@@ -29,7 +29,8 @@ Table of contents:
 
 - [Game Updates](#game-updates)
 - [Learning Material Updates](#learning-material-updates)
-- [Library & Tooling Updates](#library-tooling-updates)
+- [Library & Tooling Updates](#library--tooling-updates)
+  - [[hexasphere] v1.0](#hexasphere-v10)
```

Comment:
```suggestion
```
The table of contents usually only has the main headings in it, as it'd be super long otherwise (is this right, @ozkriff?)

Reply:
Ah, alright. I didn't write that manually, and was surprised when vscode added it automatically. I'll remove it anyway.